config/templates/osd-metrics-exporter-csv-template.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: osd-metrics-exporter-0.0.1
```

Comment:
it is not allowed with the verions lower which is 0.0.2 :)

Reply:
This is just a template which is used to generate the actual CSV and the name gets set [like so](https://github.com/openshift/osd-metrics-exporter/blob/6f33fc791d60572742b3bc0bfc67076cbe9e7fa4/hack/generate-operator-bundle.py#L95).